src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -110,6 +110,12 @@
     @Option(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
     public int SAMPLE_SIZE=10000;
 
+    @Option(shortName = "LARGE_INTERVALS", doc = "An interval list file that contains contig-sized locations of the positions to assess. It is important that " +
+            "the intervals be chosen so that they won't have overlapping reads that extend beyond the boundaries of the interval. For smaller sites use " +
+            "CollectWgsMetricsFromSampledSites.",
+            optional = true)
```

Comment:
I think I would prefer to call this just INTERVALS and change the doc text to something like this:

An interval list file that contains the positions to restrict the assessment.  Please note that all bases of reads that overlap these intervals will be considered, even if some of those bases extend beyond the boundaries of the interval.   The ideal use case for this argument is to use it to restrict the calculation to a subset of (whole) contigs.  To restrict the calculation just to individual positions without overlap, please see CollectWgsMetricsFromSampledSites.

What do you guys think?